.github/workflows/testacc.yml

```diff
@@ -11,6 +11,9 @@ on:
       - main
     paths-ignore:
       - 'README.md'
+  schedule:
+    - cron: '30 21 * * 1'  # Runs every Monday at 2:30PST for stage environment
```

Comment:
this cron expression is for 9:30 PM every monday - is this or the comment wrong?

Reply:
its UTC